src/bun.js/bindings/helpers.h

```diff
@@ -335,6 +326,16 @@ static JSC::JSValue getRangeErrorInstance(const ZigString* str, JSC__JSGlobalObj
     return JSC::JSValue(result);
 }
 
+static const JSC::Identifier toIdentifier(ZigString str, JSC::JSGlobalObject* global)
+{
+    if (str.len == 0 || str.ptr == nullptr) {
+        return JSC::Identifier::EmptyIdentifier;
+    }
+    WTF::String wtfstr = Zig::isTaggedExternalPtr(str.ptr) ? toString(str) : Zig::toStringCopy(str);
```

Comment:
What was the reason for this change?

Reply:
Previously it was just turning the raw bytes into JSC::Identifier. Which meant that any utf8 string would get turned into latin1